README.md

```diff
@@ -3,6 +3,22 @@
 This is the home of Tabzilla, the universal tab you can add to your website. Below
 follows simple instructions on how to get, and use Tabzilla.
 
+## Upgrading to the new Tabzilla
+
+Before you switch to the new Tabzilla, there are a couple of caveats you need to be
+aware of.
+
+- Tabzilla no longer needs any fonts bundled with it so, we are no longer including
+fonts in the CSS.
+- As this version of Tabzilla is static, we have no need for JavaScript and as such,
+we no longer inject jQuery if it not already available.
```

Comment:
Technically, it was correct with `"if not already available"`, as we only inject it if it's not there (or the version is too old)